docs/detections/rules-ui-create.asciidoc

```diff
@@ -82,6 +82,15 @@ When you use a saved query, the *Load saved query "_query name_" dynamically on
 * Deselect this to load the saved query as a one-time way of populating the rule's *Custom query* field and filters. This copies the settings from the saved query to the rule, so you can then further adjust the rule's query and filters as needed. If the saved query is later changed, the rule will not inherit those changes.
 
 . (Optional, https://www.elastic.co/pricing[Platinum or higher subscription] required) Use *Suppress alerts by* to reduce the number of repeated or duplicate alerts created by the rule. Refer to <<alert-suppression>> for more information.
++
+
+////
+The following steps are repeated across multiple rule types. If you change anything 
+in these steps or sub-steps, apply the change to the other rule types, too.
+////
+. (Optional) Create a list of **Required fields** that the rule needs to function. This list is informational only, to help users understand the rule; it doesn't affect how the rule actually runs.
+.. Click **Add required field**, then select a field from the index patterns or data view you specified in the rule's **Source** above. You can also start typing a field's name to find it faster, or type in an entirely new custom field.
```

Comment:
> ...the index patterns or data view you specified in the rule's **Source** above.

Step 2 doesn't explicitly mention **Source** as a section name, so using **Source** when referring to that step here could be confusing (especially in sections that don't have a screenshot showing the **Source** section). Maybe we could say "...the index patterns or data view you specified in step 2." instead? That would also avoid using directional language.

Reply:
@natasha-moore-elastic Good point. I also try to avoid referring to specific step numbers, since they're prone to change, but we can probably work around that too. How about this:
```suggestion
.. Click **Add required field**, then select a field from the index patterns or data view you specified for the rule. You can also start typing a field's name to find it faster, or type in an entirely new custom field.
```

Reply:
Sounds good!

Reply:
Done, for both serverless and classic content, in https://github.com/elastic/security-docs/pull/5287/commits/b8830267b7aa75cc53899c9e050e3f42d035d15c